### PR TITLE
EZP-29452: Rendering unused panels in Universal Discovery

### PR DESCRIPTION
**Pull request by sunpietro**
https://jira.ez.no/browse/EZP-29452

## For QA
The UDW performance should be increased with changes from this PR.

### CODE REVIEW

File: src/modules/universal-discovery/universal.discovery.module.js [hunk 1]
```diff
@@ -794,8 +794,14 @@ export default class UniversalDiscoveryModule extends Component {
...
         const { activeTab, maxHeight, contentTypesMap } = this.state;
+        const isVisible = activeTab === item.id;
+
+        if (!isVisible) {
+            return null;
+        }
+
         const attrs = {
-            isVisible: activeTab === item.id,
+            isVisible: isVisible,
```
Comment (tischsoic) [inline src/modules/universal-discovery/universal.discovery.module.js]: We can do: `isVisible: isVisible,` -> `isVisible,`

**Comment by sunpietro** [comment]
ping @tischsoic

**Review by tischsoic — APPROVED** [review]

**Review by dew326 — APPROVED** [review]

**Review by micszo — APPROVED** [review]

**Comment by lserwatka** [comment]
@sunpietro you can merge it up to master

**Comment by sunpietro** [comment]
@lserwatka it has just been merged up to merged up to the master

**Pull request merged**